app/serializers/user_serializer.rb

```diff
@@ -0,0 +1,10 @@
+class UserSerializer < ActiveModel::Serializer
+  embed :ids
+
+  attributes :id, :name, :role, :token, :username,
+             :thumbnail_url, :email
+
+  has_one  :cohort
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.